data/human/deep missions.txt

```diff
@@ -2431,3 +2431,320 @@ event "battle in arneb end"
+					goto yourself
+				`	(Let Helm Answer for you.)`
+					goto helm
+			label yourself
+			`	You smile somberly at Thorleif. "I was a friend of Skaldgar. I gave him a ride a few times, and, most recently, was entrusted with his stone."`
+					goto stone
+			label helm
+			`	Helm puts a hand on Thorleif's shoulder. "This here is <first>, they've come a long way to see you, and to bring you Skaldgar's stone."`
+			label stone
+			`	Thorleif tenses up. "Never speak that name!" His eyes narrow. "Helm, I want nothing to do with that stone, this person," Thorleif says, gesturing toward you, "or whatever else that old man wanted or cared about."`
+			`	Thorleif glances toward the horizon as he clenches and unclenches his fists, taking a few slow breaths, regaining his composure. "I realize you have traveled far and would not be able to get back to the mainland before night. So you may stay in the guest quarters, and leave first thing in the morning. I will have no one say that Thorleif was inhospitable."`
+			choice
+				`	(Keep quiet and follow Helm towards the guest rooms.)`
+					goto rooms
+				`	(Try to say something on Skaldgar's behalf.)`
+					goto behalf
+			label behalf
+			`	"Wait," you say before Thorleif can stomp off. "I do not, could not, know all that has happened here, that has gone on between you and your father. But I know that every time I met him he clearly regretted whatever happened, and constantly spoke of his mistakes. He is gone now, mistakes and all, and this stone.. I don't think it represents forgiveness, just, acceptance of who he was and that now he is gone."`
+			`	Thorleif turns to you, his face flashing red, and speaks through clenched teeth, "I have no father. I have no memories of such a man, and I will make no effort to remember someone who didn't stick around long enough to know me. Now, go to your rooms and leave me be!"`
+			label rooms
+			`	You quietly follow Helm to two small, sparsely decorated rooms, clearly intended for guest use. Helm clears his throat, "This may surprise you, but that actually went better than I thought it would."`
+			`	Helm then turns his attention to a small window and the darkening horizon beyond it. "Dark clouds are gathering, I fear we may be here longer than a single night."`
+			choice
+				`	(You head to bed as the first rains begin to patter against the roof.)`
+			`	You awake to riotous rain pelting hard against the walls. As Helm predicted a powerful storm blew in over the night, and the floating island found itself besieged by savage weather. By midmorning the walls are starting to creak as the howling wind bashes against it unceasingly.`
+			`	You hear a door open and struggle to close. Then the sound of heavy sloshing footsteps. Thorleif, soaked to the bone, stands before you and Helm in the main room, "It's a Fimbulhrithr," Thorleif exclaims, "One of the worst I've yet seen. If we do not act quickly it could well flatten everything on this isle. Will you help me?"`
```

Comment:
```suggestion
			`	You hear a door open and struggle to close, followed by the sound of heavy sloshing footsteps. Thorleif, soaked to the bone, stands before you and Helm in the main room. "It's a Fimbulhrithr," Thorleif exclaims, "One of the worst I've yet seen. If we do not act quickly it could well flatten everything on this isle. Will you help me?"`
```
Personal preference. With just the correction below:
```suggestion
			`	You hear a door open and struggle to close. Then the sound of heavy sloshing footsteps. Thorleif, soaked to the bone, stands before you and Helm in the main room. "It's a Fimbulhrithr," Thorleif exclaims, "One of the worst I've yet seen. If we do not act quickly it could well flatten everything on this isle. Will you help me?"`
```